doc/sphinx/userguide/faq.md

```diff
@@ -1,12 +1,227 @@
+chemical reactions. In contrast, it is common for tabulated data for a single pure
+substance to use a different reference state, such as CO₂ where the enthalpy may be
+referenced to the saturated liquid at 273.15 K.
+:::
+
+:::{dropdown} Why doesn't the heat capacity calculated by Cantera match what I get from CEA?
+:color: primary
+
+Cantera’s definition of $c_p$ is the partial derivative of enthalpy with respect to
+temperature, with pressure and composition held constant. The value returned by CEA is
+the derivative while holding the system at chemical equilibrium. The former definition
+is the one that appears in governing equations for multi-species systems, such as
+[well-stirred reactors](/reference/reactors/ideal-gas-constant-pressure-reactor). While
+the latter property is of interest in certain physical situations, it is of somewhat
+limited use computationally, since it is *only* defined for mixtures at equilibrium.
+
+You can use Cantera to compute a value like what’s returned by CEA by computing $dh/dT$
+using a finite difference method where the mixture is equilibrated at constant $T$ and
+$P$ at each point before calculating the enthalpy.
+[`sound_speed.py`](/examples/python/thermo/sound_speed) presents a related example.
+
+CEA provides a value consistent with Cantera's definition which it calls the $c_p$ "with
+frozen reactions."
+:::
+
+:::{dropdown} Why doesn't changing reactions affect the equilibrium state calculated by Cantera? How does Cantera find the equilibrium state even when no reactions are defined?
+:color: primary
+Chemical equilibrium is not necessarily defined in terms of any reactions. For whatever
+set of species are allowed in a phase or multi-phase mixture, it is defined as the
+composition that minimizes the Gibbs free energy (for equilibrium at constant
+temperature and pressure). This composition may be different from what you would get,
+for example, from integrating a reactor network using a set of irreversible chemical
+reactions or a set of reactions that does not form a complete basis set for the defined
+species.
+:::
+
+## Reactor Networks
+
+:::{dropdown} How do I set the residence time for a reactor?
+:color: primary
+
+Cantera defines reactor flows in terms of inlet and outlet mass flow rates, with
+residence time being an output that can be calculated:
+
+$$ t_\t{res} = \frac{m_\t{combustor}}{\dot{m}_\t{in}} $$
+
+To achieve a specified residence time, you can define the mass flow rate to be a
+function of the combustor state. For example, the following approach is used in the
+example [`combustor.py`](/examples/python/reactors/combustor):
+
+```py
+def mdot(t):
+    return combustor.mass / residence_time
+
+inlet_mfc = ct.MassFlowController(inlet, combustor, mdot=mdot)
+outlet_mfc = ct.PressureController(combustor, exhaust, primary=inlet_mfc, K=0.01)
+```
+:::
+
+:::{dropdown} How do I understand errors from ReactorNet that give a list of "components with largest weighted error estimates"?
```

Comment:
```suggestion
:::{dropdown} How do I understand errors from ReactorNet that give a list of "components with largest weighted error estimates"?
```
```suggestion
:::{dropdown} How do I understand errors from `ReactorNet` that give a list of "components with largest weighted error estimates"?
```

Reply:
For reasons I don't understand, `<code>` blocks in the title of a dropdown work very badly:
![image](https://github.com/Cantera/cantera/assets/644977/56ffb3d9-f249-448e-8549-0b17ff194879)